src/main/java/wolox/training/model/User.java

```diff
@@ -0,0 +1,110 @@
+package wolox.training.model;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+@Entity
+@Table(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false)
+    private String username;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private LocalDate birthdate;
+
+    @Column(nullable = false)
+    @ManyToMany(cascade = CascadeType.ALL)
+    @JoinTable(name = "users_book",
```

Comment:
JoinTable annotation is unnecessary unless you want to override the naming of the table and/or it's columns. Otherwise, it get's generated like this by default :)

Reply:
Great! Done